src/v2/view-builder/views/DeviceChallengePollView.js

```diff
@@ -61,10 +61,21 @@ const Body = BaseForm.extend(Object.assign(
         this.doLoopback(deviceChallenge.domain, deviceChallenge.ports, deviceChallenge.challengeRequest);
         break;
       case 'CUSTOM_URI':
-        this.title = loc('customUri.title', 'login');
-        this.subtitle = loc('customUri.subtitle', 'login');
+        this.title = loc('oktaVerify.button', 'login');
         this.add(View.extend({
-          template: hbs`{{{i18n code="customUri.content" bundle="login"}}}`
+          template: hbs`
+            <div class="skinny-content">
+              {{{i18n code="customUri.required.content.p1" bundle="login"}}}
+            </div>
+            <div class="skinny-content">
```

Comment:
we can remove the extra `div` with same class and instead use <p> tags ?

Reply:
This is to put some spacing between the two divs.

Reply:
yes, the <p> tags will does it for you.

Reply:
`p` does not add the spacing for you. With `div`, no new style needs to be added. Other than making the UI more semantic, what's the significance of using `p` here?

Reply:
I think it's good enough reason to use p, proper semantics. If they were different classes i would have not asked to change it.

use p for paragraphs, div for containers.